test/EndToEndTests/Tests/Client/AsynchronousTests/AsynchronousQueryTests.cs

```diff
@@ -294,21 +295,25 @@ public void ExecuteBatchWithSaveChangesOptionsReturnsCorrectResults()
             this.EnqueueTestComplete();
         }
 
-#if !PORTABLELIB && !NETCOREAPP1_0
         /// <summary>
         /// CancelRequest for Batch Requests
         /// </summary>
-        // github issuse: #896
-        // [Fact, Asynchronous]
+        [Fact, Asynchronous]
         public void CancelBatchRequestTest()
         {
             var context = this.CreateWrappedContext<DefaultContainer>().Context;

```

Comment:
Is the `context` a `DataServiceContext`?

Reply:
yes

Reply:
Use the concrete type instead of var